spec/espec_phoenix_extend.ex

```diff
@@ -1,5 +1,5 @@
 defmodule ESpec.Phoenix.Extend do
-  def model do
+  def domain do
```

Comment:
We don't have models as is, so why not :)

Reply:
At first sight, I wanted to write something like 'Yep, why not to fuck this shit up', but I desided not to do it. So, well done!